### DISCUSSION TITLE
建议标准库文档翻译加上 rustc 版本，毕竟 rustc 版本变动标准库文档也是会变动很多的

### DISCUSSION THREAD

**Discussion by pymongo** [rust-lang-cn/rustwiki.org#6] [General]
现在网页上所有已翻译的文档都写个 100% 已翻译，其实写了等于没写，没什么信息量

还不让放一个翻译的版本对应的 rustc 版本是多少

**Comment by aaranxu** [comment]
不错的建议，网站后续会考虑版本信息，特别是重点维护的文档，对于可能时效性过时的文档也注明（主要是精力有限，有些文档后期更新可能落下啦）。

**Comment by aaranxu** [comment]
另外我们翻译小组也计划进一步完善文档的版本的发布，类似于官网那样，每个稳定版都会推出对应的访问地址，如 url/1.58.0/book 这样。

**Reply by percy507** [reply]
辛苦啦👍👍👍
